src/font.rs

```diff
@@ -67,8 +67,10 @@ impl<'a> Font<'a> {
     ///  - `text`: text to render.
     ///  - `row`: x (Left/Right align) or y (Up/Down align) offset where to stop
     ///    rendering.
-    ///  - `wh`: the size of each character in X & Y dimensions.
     ///  - `text_align`: how the text is aligned
+    /// 
+    ///  Returns an iterator which generates the path from characters (see [`TextPathIterator`])
+    ///  and a number indicating what was leftover
```

Comment:
The leftover number should be documented better, but I'm not entirely sure what it's measured in. Maybe provide an example of how to use it?

Reply:
It's the number of characters not rendered.  I can add an example on how to use it.